enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -1,113 +1,115 @@
-          <p>${ consent_request_prompt } ${ policy_link_template.format(start_link='<a href="#consent-policy-dropdown-bar" class="policy-dropdown-link background-input" id="policy-dropdown-link">', end_link='</a>') }</p>
-          <p>${ requested_permissions_header }
-          <ul class="consent-items">
-            % for permission in requested_permissions:
-            <li>${ permission }</li>
-            % endfor
-          </ul>
+          {% autoescape off %}
+            <p>{{ consent_request_prompt }} {{ policy_link_template }}</p>
+          {% endautoescape %}
+
+          <p>{{ requested_permissions_header }}
+            <ul class="consent-items">
+              {% for permission in requested_permissions %}
+                <li>{{ permission }}</li>
+              {% endfor %}
+            </ul>
           </p>
         </div>
+
         <div class="consent-input-container login-register">
           <form name="data-sharing-consent" method="POST" id="data-sharing">
+            {% csrf_token %}
+
             <input class="background-input data-consent-checkbox" type="checkbox" name="data_sharing_consent" id="data-consent-checkbox" value="True"/>
             <p class="agreement-text">
-              <label for="data-consent-checkbox">${agreement_text}</label>
+              <label for="data-consent-checkbox">{{ agreement_text }}</label>
             </p>
-            <button type="submit" class="background-input consent-agreement-button" id="consent-button" disabled>${ continue_text }</button>
-            <button type="button" class="failure-link background-input" id="failure-link">${ abort_text }</a>
-            <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }" />
-            % if course_specific and course_id:
-            <input type="hidden" name="course_id" value="${ course_id }" />
-            % endif
-            % if redirect_url:
-            <input type="hidden" name="redirect_url" value="${redirect_url}" />
-            % endif
-            % if failure_url:
-            <input type="hidden" name="failure_url" value="${failure_url}" />
-            % endif
-            % if enrollment_deferred:
-            <input type="hidden" name="enrollment_deferred" value="true" />
-            % endif
+            <button type="submit" class="background-input consent-agreement-button" id="consent-button" disabled>{{ continue_text }}</button>
+            <button type="button" class="failure-link background-input" id="failure-link">{{ abort_text }}</button>
+
+            {% if course_specific and course_id %}
+              <input type="hidden" name="course_id" value="{{ course_id }}" />
+            {% endif %}
+
+            {% if redirect_url %}
+              <input type="hidden" name="redirect_url" value="{{ redirect_url }}" />
+            {% endif %}
+
+            {% if failure_url %}
+              <input type="hidden" name="failure_url" value="{{ failure_url }}" />
+            {% endif %}
```

Comment:
The way these are used in the code it seems things would break if we remove these conditionals. Empty value checks are not present for these in the view.